packages/astro/test/types/astro-config-virtual-module.ts

```diff
@@ -0,0 +1,15 @@
+import { expectTypeOf } from "expect-type";
+import { describe, it } from "node:test";
+import type { ServerDeserializedManifest, ClientDeserializedManifest } from "../../dist/types/public";
```

Comment:
This is causing tests to fail on main. It needs a file extension

Reply:
Thank you Matt for the heads up